llama-index-integrations/vector_stores/llama-index-vector-stores-duckdb/pyproject.toml

```diff
@@ -0,0 +1,50 @@
+[build-system]
+requires = ["poetry-core"]
+build-backend = "poetry.core.masonry.api"
+
+[tool.codespell]
+check-filenames = true
+check-hidden = true
+# Feel free to un-skip examples, and experimental, you will just need to
+# work through many typos (--write-changes and --interactive will help)
+skip = "*.csv,*.html,*.json,*.jsonl,*.pdf,*.txt,*.ipynb"
+
```

Comment:
Hey I know this wasn't yet specified in our CONTRIBUTING guidelines, but nevertheless can you please add a section for `[tool.llamahub]` the one found [here](https://github.com/run-llama/llama_index/blob/8431e7baaeb8155c33a7db328fce79bd7d533feb/llama-index-integrations/vector_stores/llama-index-vector-stores-astra/pyproject.toml#L10)

Reply:
fixed this.